index.html

```diff
@@ -36,6 +36,168 @@ <h2 class="connect-link"> <a href="#">LET'S CONNECT</a></h2>
     </ul>
     </div>
 </section>
-  
+<section class="work-section">
+    <div class="tonic-container">
+        <div><img src="images/tonic-portfolio.png" alt="tonic"></div>
```

Comment:
- [ ] Your page's background image is not matched with the Figma design. Please work on it. :+1: 

- [ ] The cards are not aligned properly. Kindly fix it.

Your design             |  Figma Design
:-------------------------:|:-------------------------:
![y1](https://user-images.githubusercontent.com/70483835/164862006-12034cf8-a2b9-4823-9f7d-c14c5eed0acb.png) | ![f1](https://user-images.githubusercontent.com/70483835/164861937-249e2e69-e260-46af-8184-4aceb3949ab8.png)